store/root/store.go

```diff
@@ -158,6 +161,8 @@ func (s *Store) GetLatestVersion() (uint64, error) {
 }
 
 func (s *Store) Query(storeKey string, version uint64, key []byte, prove bool) (store.QueryResult, error) {
+	defer telemetry.MeasureSince(time.Now(), "root_store", "query")
```

Comment:
could store define a quick telemetry interface that allows time.Now() to be turned off. Dev mentioned there was some overhead to this

Reply:
The only way to do this that I can think of is:

```go
func (s *Store) LoadVersion(version uint64) error {
	if s.telemetryEnabled {
		defer telemetry.MeasureSince(time.Now(), "root_store", "load_version")
	}

	// ...
}
```

This is because you need to get the start time during the start time of the method. Even with an interface, you'll need to still call `time.Now()` in the method itself.